lib/README.md

```diff
@@ -212,6 +212,11 @@ The file structure is designed to make this selection manually achievable for an
   For this scenario, it can be set as `ZDICT_QSORT=ZDICT_QSORT_C90`.
   Other selectable suffixes are `_GNU`, `_APPLE`, `_MSVC` and `_C11`.
 
+- The build macro `ZSTD_COMPRESS_FAST` can be defined for fast lazy evaluation
```

Comment:
The macro name is not specific enough. The name implies that this is about the "fast mode", while the changes are actually impacting the `ZSTD_lazy` strategy. Consider a more accurate name, or better, if we can find just a better trade off, maybe there would be no need for another build macro.